pantheon/src/main/java/tech/pegasys/pantheon/cli/PantheonCommand.java

```diff
@@ -202,62 +198,86 @@ public RpcApi convert(final String name) throws RpcApisConversionException {
   private final Integer maxTrailingPeers = Integer.MAX_VALUE;
 
   @Option(
-    names = {"--banned-nodeids"},
+    names = {"--banned-node-ids", "--banned-node-id"},
+    paramLabel = MANDATORY_NODE_ID_FORMAT_HELP,
     description = "A list of node IDs to ban from the p2p network.",
     split = ",",
     arity = "1..*"
   )
   private final Collection<String> bannedNodeIds = new ArrayList<>();
 
-  // TODO: Re-enable as per NC-1057/NC-1681
-  //  @Option(
-  //    names = {"--sync-mode"},
-  //    paramLabel = MANDATORY_MODE_FORMAT_HELP,
-  //    description =
-  //        "Synchronization mode (Value can be one of ${COMPLETION-CANDIDATES}, default:
-  // ${DEFAULT-VALUE})"
-  //  )
-  private final SyncMode syncMode = DEFAULT_SYNC_MODE;
```

Comment:
It would be quite helpful if we kept this as-is - I have another open PR that starts using it again. :)

Reply:
It's me, deciding to remove dead code just before it gets to life gain after month being dead...